docs/Feature Documentation/Audio and Video Streaming Guide/index.md

```diff
@@ -31,54 +21,46 @@ Comment out the following lines in `app/model/sdl/Abstract/Model.js`:
 
 ### GSTREAMER Setup
 
-First, we must determine what gstreamer command works in your environment
+It is easier to determine which gstreamer video sink will work in your environment by testing with a static file. This can be done by downloading [this file](https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/HT1425/sample_iPod.m4v.zip) and trying the following command.
 
-Start by finding a raw h.264 file ([Example](https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/HT1425/sample_iPod.m4v.zip)) and determine which of the these gstreamer commands successfully plays the example video
+Common values for sink:
+- ximagesink (x visual environment sink)
+- xvimagesink (xv visual environment sink)
+- cacasink (ascii art sink)
 
-- Ubuntu 14.04+
-```
-gst-launch-1.0 filesrc location=/path/to/h264/file ! decodebin ! videoconvert ! xvimagesink sync=false
 ```
-or
-```
-gst-launch-1.0 filesrc location=/path/to/h264/file ! decodebin ! videoconvert ! ximagesink sync=false
+gst-launch-1.0 filesrc location=/path/to/h264/file ! decodebin ! videoconvert ! <sink> sync=false
 ```
 
-If you're using tcp, you can connect the stream directly with your phone's IP address using
+If you're streaming video over TCP, you can point gstreamer directly to your phone's stream using
 ```
 gst-launch-1.0 tcpclientsrc host=<Device IP Address> port=3000 ! decodebin ! videoconvert ! ximagesink sync=false
 ```
 
 ## Pipe Streaming
 
 ### Configuration (smartDeviceLink.ini)
-In the Core build folder, open `bin/smartDeviceLink.ini` and the make the following changes:
+In the Core build folder, open `bin/smartDeviceLink.ini` and ensure the following values are set:
```

Comment:
This same sentence needs to be changed in several places in the guide

Reply:
Thanks! Updated in https://github.com/smartdevicelink/sdl_core_guides/pull/82/commits/c7b99869d92a546102ef6fd25d556729657e72d6